source/mainnet/net/resources/glossary.rst

```diff
@@ -371,7 +379,7 @@ Invoke is also the act of triggering a receive function in a smart contract from
 
 .. _glossary-leader-election:
 
-Leader Election
+Leader Election (to be updated?)
```

Comment:
Yes, this is outdated and completely different in the new consensus.

Reply:
I have added some of the text that was previously under Round because it seems more relevant here. And I have tried to restructure it in a way that makes sense.